maintainers/maintainer-list.nix

```
@@ -290,6 +290,12 @@
     githubId = 335271;
     name = "James Alexander Feldman-Crough";
   };
+  afontain = {
```

```suggestion
  necessarily-equal = {
```
could you use your github name there, so others can just use it to mention you here? it's always an issue when it's not identical

---

dunno about that, I'm known as afontain about everywhere *but* github. And there is a `github` attribute for this very reason?

---

people just copy and paste the name from the derivation and use it to mention the maintainer. if that does not work i often don't bother to mention the maintainer. it's too much work to look in the `maintainer-list.nix`

maybe you can change your github name to something you like to use more? if there is no feature, ask the support. they are pretty helpful

---

« **About username changes**

You can change your username to another username that is not currently in use. If the username you want is not available, you'll see information about whether you can request the username to be released when you type in the desired username.

If the username is not eligible for release and you don't hold a trademark for the username, you can choose another username or keep your current username. GitHub Support cannot release the unavailable username for you. For more information, see "Changing your username." » — https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-user-account-settings/changing-your-github-username

looks like nope